flag/service/installation/guest/kubernetes/api/auth/provider/oidc/oidc.go

```diff
@@ -0,0 +1,8 @@
+package oidc
+
+type OIDC struct {
+	ClientID      string
+	IssuerUrl     string
```

Comment:
Nitpick: URL should be in caps.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms